src/aat/resources/features/F-036/S-202.td.json

```diff
@@ -0,0 +1,44 @@
+{
+  "_guid_": "S-202",
+  "_extends_": "F-036_Base_Test_Data",
+  "title": "must return 401 when request does not provide valid authentication credentials",
+  "specs": [
+    "an active profile in CCD",
+    "does not provide valid authentication credentials in CCD",
+    "has a 403 Forbidden code"
+  ],
+  "users": {
+    "invokingUser" : {
+      "username": "befta.citizen.3@gmail.com",
+      "password": "InvalidPassword"
```

Comment:
I believe the userID and password should be a valid combination so that user can be authenticated. It's the authorisation which should fail.